src/app/styles/components/_variables.scss

```diff
@@ -6,3 +6,10 @@ $headerbar-background: #466BC1;
 $button-hover-color: #3859A5;
 $button-padding: 12px;
 $logout-button-color: #FFFFFF;
+
+// Introduction stage variables
```

Comment:
I discussed about this with Shreyans, @yukie1412 . He has made the scss structure changes in one of his open PRs, I have made the changes to exclude importing the old file.